### PR TITLE
Changes query endTs and lookback from micros to millis

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Time units of query parameters endTs and lookback are now milliseconds
as opposed to microseconds, the grain of Span and Annotation timestamp.
Milliseconds is a more familiar and supported granularity for query,
index and windowing functions.

This change does not change implementation schema. Span stores who use
custom indexes (like cassandra and redis) probably want to update to use
millis now.

See #857

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
manual testing of the web app works with queries and dependency graph

File: zipkin-thrift/src/main/thrift/com/twitter/zipkin/zipkinDependencies.thrift [hunk 1]
```diff
@@ -21,16 +21,16 @@ struct DependencyLink {
...
-  /** calls made during the duration (in microseconds) of this link */
+  /** calls made during the duration of this link */
   4: i64 callCount
   # histogram?
 }
 
 /* An aggregate representation of services paired with every service they call. */
 struct Dependencies {
-  /** microseconds from epoch */
+  /** milliseconds from epoch */
```
Comment (yurishkuro) [inline zipkin-thrift/src/main/thrift/com/twitter/zipkin/zipkinDependencies.thrift:31]: Added https://github.com/openzipkin/zipkin-dependencies-spark/issues/3

Comment (codefromthecrypt) [inline zipkin-thrift/src/main/thrift/com/twitter/zipkin/zipkinDependencies.thrift:31]: Thx for logging this

**Comment by yurishkuro** [comment]
Fixed #857

**Pull request merged**